### ISSUE TITLE
Notification for new repository creation

### ISSUE THREAD

**Issue by danlopez** [integrations/microsoft-teams#85]
**Is your feature request related to a problem? Please describe.**
I'm the organization owner for a large organization with clear repository creation standards. I'd like to know when new repos are created so I can reach out to developers and remind them about things like a clear README.md, license, etc. 

**Describe the solution you'd like**
I'd like the able to subscribe to the repository:created webhook and receive Teams messages. 

**Describe alternatives you've considered**
N/A
**Additional context**
N/A

**Comment by danlopez** [comment]
@ashokirla is this being considered at all?

**Comment by ashokirla** [comment]
@danlopez yes, Notifications for Repo operations is in our backlog. We will look into it next quarter.

**Comment by ku1918** [comment]
+1 for this request
